sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubPermission.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.WebPubSub
+{
+    /// <summary>
+    /// Web PubSub Authentication Policy.
+    /// </summary>
+    public enum WebPubSubPermission
+    {
+        /// <summary>
+        /// Permission to send messages to a group.
+        /// </summary>
+        SendToGroup,
```

Comment:
Nit - consider assigning these numeric values now so nobody breaks things by accidentally inserting a value in the middle.